### PR TITLE
REL v0.0.12

### PR DESCRIPTION
**Pull request by andyfaff**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,31 +12,30 @@ source:
...
 build:
-  number: 1
+  number: 3
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - toolchain
     - python
-    - numpy x.x
```
Comment (ocefpaf) [inline recipe/meta.yaml:22]: I don't think you can do that b/c this package seems to rely on numpy's ABI. See https://github.com/refnx/refnx/blob/master/setup.py#L153

Also, without [re-rendering](https://conda-forge.org/docs/conda_smithy.html), this change is not in effect.

Last but not least the `numpy x.x` is deprecated. See https://conda-forge.org/docs/meta.html#building-against-numpy for a better way to do that.

Comment (andyfaff) [reply recipe/meta.yaml:22]: Thanks for these comments. Everytime I come to update the recipe it seems that the process has changed significantly, so it's a large learning process again.

Comment (ocefpaf) [reply recipe/meta.yaml:22]: conda-forge actually evolves quite slowly if compared to similar systems :wink: 

But I get that. The only way to keep in touch is to hang out in our [gitter](https://gitter.im/conda-forge/conda-forge.github.io) channel.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml:22]: PS: I am using https://conda-forge.org/docs/webservice.html to re-render this recipe for you.

**Comment by andyfaff** [comment]
@ocefpaf I thought this PR was in a rerendered state. Is there anything else need to do here?

**Comment by ocefpaf** [comment]
> @ocefpaf I thought this PR was in a rerendered state.

Nope. See what the re-rendering bot added in https://github.com/conda-forge/refnx-feedstock/pull/4/commits/5395099219d03f7f90823119d304097c2c1059ec

BTW you can use those webservices too and save time instead re-rendering locally :wink: 

>  Is there anything else need to do here?

If the CIs passes no. Fingers crossed.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,31 +12,30 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 5
```
Comment (ocefpaf) [inline recipe/meta.yaml:15]: You can bump it to `2`. No need to skip to `5`.
It does not matter much but some users may wonder where are builds 2-4

**Comment by andyfaff** [comment]
The penultimate appveyor run timed out. I then restarted it just now, but it got cancelled for some reason, I don't know why.
When the package tests are normally rum on appveyor they only take 6 mins each,so I don't know what's going on.

**Comment by ocefpaf** [comment]
> The penultimate appveyor run timed out. I then restarted it just now, but it got cancelled for some reason, I don't know why.

Those jobs are near the time limit. You can probably merge the PR, to "cash" Linux and OS X, and re-start AppVeyor on master to get the builds on the channel.

**Pull request merged**